### DISCUSSION TITLE
App Name

### DISCUSSION THREAD

**Discussion by med4kat** [Nerds-Who-Code/Mental-Health-Tracker#11] [Polls]
May I please ask you to vote on the name by the end of 2 July?
Many thanks!

**Comment by xintamosaik** [comment]
1. Feelsify
2. Moodkeeps

**Reply by med4kat** [reply]
Could you please choose one?

**Reply by ooddaa** [reply]
he chose Feelsify =) I like Moodkeeps more tbh

**Comment by med4kat** [comment]
Moodkeeps = 32% vs Feelsify = 62%
Discussed in discord the pronunciation of Feelsify. Was resolved to go for Feelsify.
